distributionResources/log4j2.xml

```diff
@@ -21,8 +21,22 @@
             </PatternLayout>
             <Policies>
                 <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="10 MB"/>
             </Policies>
-            <DefaultRolloverStrategy max="3"/>
+            <DefaultRolloverStrategy fileIndex="min" max="3">
+                <Delete basePath="${sys:labkey.log.home}">
+                    <ScriptCondition>
+                        <!-- Use custom Java code to retain the first log file from the current session if it's about
+                        to be rotated out. It may have root cause information about problems that would otherwise be lost -->
+                        <Script name="retainFirst" language="rhino"><![CDATA[
```

Comment:
rhino, eh? Do we know it's available where it's needed?

Reply:
I don't think the "rhino" reference will be a problem. Normal installers built by TeamCity include the file from the server repo which has included [this script reference](https://github.com/LabKey/server/pull/124) for a couple of years now. These files are only included in standalone distributions.